src/command/which.rs

```diff
@@ -34,20 +63,29 @@ impl Command for Which {
             None => System::path()?,
         };
 
-        match which_in(&self.binary, Some(path), cwd) {
+        let path = match tool_path {
+            Some(tool_path) => {
```

Comment:
If `tool_path` is a `Some(_)` value, it appears this method will return on line 50, so we won't ever reach this branch of the `match`

Reply:
This whole path is extraneous; I totally forgot to rip it back out.

Reply:
Resolved in 12d7e6b0.